openapi/api.yaml

```diff
@@ -189,6 +250,18 @@ components:
       in: header
       name: X-Xsrf-Token
   schemas:
+    PostIntentRequest:
+      properties:
+        gate_id:
+          type: integer
+        intent_cc_emails:
+          type: array
+          items:
+            type: string
+    MessageResponse:
```

Comment:
MessageResponse seems very generic but I don't think this component will be used for other endpoints correct?

Reply:
There are a number of endpoints that will likely use this response structure, e.g. [FeaturesAPI](https://github.com/GoogleChrome/chromium-dashboard/blob/main/api/features_api.py#L314-L341)

Reply:
Thanks for verifying that other endpoints will use it. Makes sense then.